recipe/meta.yaml

```diff
@@ -41,11 +41,18 @@ requirements:
     # - libscotch # no 64bit support
     - libhwloc
     - metis
+    - python
+
+  run:
+    - python
+    - numpy
+    - scipy
```

Comment:
You can move all these dependencies to the test section, to avoid adding them as hard dependencies.

Reply:
nevermind, it installs packages in site-packages.